code-for-canada/project-thundercat
# [Test Content Definition - Background] Using markdown files for Background Content (3)

# Description

This PR is introducing the usage of markdown files instead of LOCALIZE strings to populate the Information about the Organizational Structure part of the background content.

However, we still have to figure out how to introduce markdown files for the image description of this section.

## Type of change

- New feature (non-breaking change which adds functionality)

## Screenshot

**Old Version - localize strings (EN):**

![image](https://user-images.githubusercontent.com/23021242/60180882-02b67280-97ef-11e9-8e38-e6e4c7d211fc.png)

**Old Version - localize strings (FR):**

![image](https://user-images.githubusercontent.com/23021242/60180897-09dd8080-97ef-11e9-814f-1d3bb0340ef2.png)

**New Version - markdown files (EN):**

![image](https://user-images.githubusercontent.com/23021242/60180828-e581a400-97ee-11e9-8982-9f11503dd1d4.png)

**New Version - markdown files (FR):**

![image](https://user-images.githubusercontent.com/23021242/60180846-efa3a280-97ee-11e9-8268-10a1ebc0074a.png)

# Testing

Manual steps to reproduce this functionality:

1.  Launch the eMIB Sample Test.
2.  Start the test.
3.  Select the _Background_ tab.
4.  Make sure that you get the content when you select _Organizational Structure_ on the side navigation menu.

# Checklist
